src/screens/ImportSeedPhraseSheet.js

```diff
@@ -142,13 +142,10 @@ const Sheet = styled(Column).attrs({
   z-index: 1;
 `;
 
-export default function ImportSeedPhraseSheet() {
+export function useImportingWallet() {
```

Comment:
would be nice to keep hooks in hooks/ until we have consensus on a nice new dir structure

Reply:
can do this after